src/main/java/it/gov/pagopa/pu/workflow/service/WorkflowServiceImpl.java

```diff
@@ -70,4 +72,28 @@ public WorkflowStatusDTO getWorkflowStatus(String workflowId) {
       throw new WorkflowInternalErrorException(e.getMessage());
     }
   }
+
+  @Override
+  public <T> T buildWorkflowStubDelayed(Class<T> workflowClass, String taskQueue, String workflowId, Duration startDelay) {
+    return workflowClient.newWorkflowStub(
+      workflowClass,
+      WorkflowOptions.newBuilder()
+        .setTaskQueue(taskQueue)
+        .setWorkflowId(workflowId)
+        .setStartDelay(startDelay)
+        .build());
+  }
+
+  @Override
+  public <T> T buildWorkflowStubScheduled(Class<T> workflowClass, String taskQueue, String workflowId, LocalDateTime dateTime) {
+    Duration startDelay = Duration.between(LocalDateTime.now(), dateTime);
+    return buildWorkflowStubDelayed(workflowClass, taskQueue, workflowId, startDelay);
+  }
+
+  @Override
+  public <T> T buildWorkflowStubScheduled(Class<T> workflowClass, String taskQueue, String workflowId, OffsetDateTime dateTime) {
+    OffsetDateTime offsetDateTimeNow = OffsetDateTime.now(ZoneId.systemDefault());
+    Duration startDelay = Duration.between(offsetDateTimeNow, dateTime.atZoneSameInstant(ZoneId.systemDefault()));
```

Comment:
Is it this really necessary?
```suggestion
    Duration startDelay = Duration.between(offsetDateTimeNow, dateTime);
```